infrastructure-formula/python/README.md

```diff
@@ -0,0 +1,26 @@
+# Python pillar helpers
+
+A Python library to be used in `#!py` pillar SLS files. It allows for rendering of formula pillars based off data in YAML datasets.
+The logic is opinionated and specific to the architecture of our code based infrastructure.
+
+## Usage
+
+
+Example pillar file `network.sls`:
+
+```
+
```

Comment:
Nitpick: the blank lines 8 and 12 are superfluous

Reply:
True, thanks